### DISCUSSION TITLE
Players cannot read books in Lecterns.

### DISCUSSION THREAD

**Discussion by Misquoth** [TownyAdvanced/Towny#4800] [Q&A] [answered]
Hi there. 

I'm trying to be able to have a Lectern with a book that can't be removed.

However, even if I open all perms for outsiders, no one can even read the book. 

using luckperms and world guard, but have not configured any area with worldguard.

How can I debug / troubleshoot to figure out why other players cannot even read the book?

**Comment by LlmDl** [comment, marked as answer]
Probably make sure lectern isn't in your switch list in the config. Otherwise file a bug ticket.
